cmd/application_config_unset.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/civo/civogo"
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var envVarName string
+
+var appConfigUnSetCmd = &cobra.Command{
+	Use:     "unset",
+	Args:    cobra.MinimumNArgs(1),
+	Short:   "Unset application config",
+	Example: "civo app config unset APP_NAME --name=foo",
+	Run: func(cmd *cobra.Command, args []string) {
+		client, err := config.CivoAPIClient()
+		if err != nil {
+			utility.Error("Creating the connection to Civo's API failed with %s", err)
+			os.Exit(1)
+		}
+
+		findApp, err := client.FindApplication(args[0])
+		if err != nil {
+			utility.Error("%s", err)
+			os.Exit(1)
+		}
+
+		config := &civogo.UpdateApplicationRequest{
```

Comment:
I don't understand how this will unset the env variable. Won't we have to loop through the existing variables, remove the one the user wants and pass that list in the API call?

Reply:
Any update on this?